content/english/anti-vegan/pests.md

```diff
@@ -0,0 +1,127 @@
+
+> Speciesism is a form of discrimination based on species membership. It involves treating members of one species as morally more important than members of other species even when their interests are equivalent.  
+>  - [Wikipedia](https://en.wikipedia.org/wiki/Speciesism)
+
+Vegans will NOT accept this deginition of pest as it is 'speciesist’. That an animal’s interests conflict with human interests is not, in itself, a reason for defining the animal as a pest or for taking action against them. Where crops are concerned alternative means such as fencing, sonics, and eco-friendly organic practices are available.  
+
+## Re-defining ‘pest’.
+
+> "A thing is right when it tends to preserve the integrity, stability and beauty of the biotic community. It is wrong when it tends otherwise.”  
+> - Aldo Leopold (1949) [The Land Ethic](https://rintintin.colorado.edu/~vancecd/phil3140/Leopold.pdf) 
+
+Aldo Leopold, the father of environmental ethics, provided a non-speciesist definition of pest when he claimed “A thing is right when it tends to preserve the integrity, stability and beauty of the biotic community. It is wrong when it tends otherwise.”  If we apply this axiom a species is a pest when, regardless of human interest, it upsets the stability of, interrupts the integrity of, or leads to the destruction of the environment.  
+
+Two New Zealand specific examples:
+
+### Are wild rabbits a pest?
+
+Biosecurity NZ lists rabbits as pests because they ‘compete with livestock, cause extensive damage to farming land, cause public nuisance… 
+
+So, yes, they are annoying to (some) humans but they are not a ‘pest’. The reasons provided by Biosecurity NZ fail under Leopold’s criteria. If wild rabbits are a pest other substantive, non-speciesist, reasons must be provided.
+
+### Are possums a pest?
+
+Possums occur in high numbers in NZ, breed fast and face very few predators. Without doubt possums have a major impact on New Zealands native ecosystems.
+
+Biosecurity NZ lists possums as pests, as, “Leaves are the main part of their diet, but possums are opportunistic omnivores. They eat buds, flowers, fruit/berries and nectar, which means they compete with native birds and reptiles for food sources. The growth and life-cycle of a tree or plant is significantly affected when all parts of it are eaten. Possums also have ‘favourites’ such as rātā or kamahi trees, leading to an even greater impact on these species.  …Possums were [also]filmed eating the eggs and chicks of kōkako …  They eat invertebrates, including weta, and are significant predators of New Zealand land snails such as Powelliphanta. They often occupy holes in tree trunks for their nests which would otherwise be used by nesting birds such as kākāriki and saddlebacks. …Researchers using nest-cameras have witnessed … kea nests invaded by stoats and possums in South Westland. ”
```

Comment:
Can we cite the Biosecurity NZ listing? (preferably via web.archive.org?)

Reply:
OK.